### PR TITLE
[Feature] Add possibility to pass initial admin credentials with env variables

### LINKED ISSUE #1597: Configure admin email/password through environment variables?
**Is your feature request related to a problem? Please describe.**
Currently on initial startup, you need to log in with the default admin email/password, then update values.

**Describe the solution you'd like**
Configure the admin email/password/username through environment variables, so there is no need to use the UI.

### PR DESCRIPTION
**Pull request by twinpixxx**
## 📃 Description
This PR adds possibility to pass admin credentials for *initial* startup. Closes #1597 
Docs repo PR: https://github.com/alexjustesen/speedtest-tracker-docs/pull/52
### ➕ Added

- Environment variables

### ✏️ Changed

- Initial migration in which the admin user is created
- .env.example file

### CODE REVIEW

**Comment by DenuxPlays** [comment]
Wouldnt make Sense to accept and already hashed password?

Services like vaultwarden already do this because its way more Secure.

**Comment by twinpixxx** [comment]
@DenuxPlays it would be overcomplicated, IMHO. By design, you (as the end user) expect the password provided in the .env file to work in the web view.
I agree that it’s more secure, but with the current implementation of user management and the process for creating the default admin password, it would feel a bit odd.

**Comment by alexjustesen** [comment]
@twinpixxx couple of minor changes, also thank you for linking to the docs PR as well 👍

File: config/app.php [hunk 1]
```diff
@@ -16,4 +16,10 @@
 
     'force_https' => env('FORCE_HTTPS', false),
 
+    'admin_name' => env('ADMIN_NAME', 'Admin'),
+
+    'admin_email' => env('ADMIN_EMAIL', 'admin@example.com'),
+
+    'admin_passwd' => env('ADMIN_PASSWD', 'password'),
```
[the remark this answers is not in the document]
Comment (twinpixxx) [reply config/app.php]: Done.

File: .env.example [hunk 2]
```diff
@@ -12,6 +12,10 @@ APP_FAKER_LOCALE=en_US
 APP_MAINTENANCE_DRIVER=file
 APP_MAINTENANCE_STORE=database
 
+ADMIN_NAME=Admin
```
[the remark this answers is not in the document]
Comment (twinpixxx) [reply .env.example]: Done. Just out of curiosity, why? Isn’t it quite logical to have all the used config variables in `.env.example`?

**Review by alexjustesen — APPROVED** [review]

**Review by alexjustesen — APPROVED** [review]

**Pull request merged**